src/main/java/com/example/demo/DemoApplication.java

```diff
@@ -10,6 +10,8 @@ public class DemoApplication implements CommandLineRunner {
     public static void main(String[] args) {
 
         //this line is written to check it from our end
+        System.out.println("one");
```

Comment:
why is this line written

Reply:
this is written to include the print

Reply:
can we remove that line

Reply:
no it should be there to print
```suggestion
        System.out.println("five");
```

Reply:
sure will make them

Reply:
made the changes